fastplotlib/layouts/_gridplot.py

```diff
@@ -415,7 +415,7 @@ def __init__(self, plot: GridPlot):
         self.flip_camera_button = Button(
             value=False,
             disabled=False,
-            icon="arrows-v",
+            icon="arrow-up",
             layout=Layout(width="auto"),
             tooltip="flip",
```

Comment:
maybe change the tooltip to "y-axis direction" or something? or if we want to be mean, "direction of the basis vector ..." :sweat_smile: